tests/models/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ if(NOT QMC_CUDA)
   simple_run_and_check(short-sho-grad_lap "${qmcpack_SOURCE_DIR}/tests/models/sho" grad_lap.xml 1 1 check_grad_lap.py)
 
   qmc_run_and_check(
-    short-grid_ext_pot-vmc
```

Comment:
this test runs 256 blocks. It is not a deterministic test

Reply:
How might it be converted to a deterministic test? Other ideas to create a unit or deterministic test that covers this memory error?

Reply:
Open an issue and ping @Hyeondeok-Shin, he can help you out

Reply:
I'm not sure a deterministic test /unit test to catch this memory error is worth the effort in this case.   This is a memory leak (I assume?) from a "put" method, which is only going to be called once per run.

Reply:
Extra motivation: ongoing test coverage in CI.